src/agentscope/studio/static/css/workstation.css

```diff
@@ -839,4 +839,35 @@ pre[class*="language-"] {
     font-size: 16px;
     line-height: 1.2;
     resize: vertical;
+}
+
+.new-action {
```

Comment:
The setting of .new-action is repeated in lines 663 and lines 844. We should keep the code clean and tidy.